docs/en_US/Release.md

```diff
@@ -1,5 +1,48 @@
 # ChangeLog
 
+## Release 1.6 - 5/26/2020
+
+### Major Features
+
+#### New Features and improvement
+* Improve IPC limitation to 100W
+* improve code storage upload logic among trials in non-local platform
+* support "__version__" for SDK version
+
+### Web UI
+*  Show trial error message
+*  finalize homepage layout
+*  Refactor overview's best trials module
+*  Remove multiphase from webui
+*  add tooltip for trial concurrency in the overview page
+*  Show top trials for hyper-parameter graph
+
+### Examples
+* TF 2.0 examples
+
+### HPO Updates
+* Improve PBT on failure handling and support experiment resume for PBT
+
+### NAS Updates
+* Use OrderedDict for LayerChoice
+* Prettify the format of export
+* Replace layer choice with selected module after applied fixed architecture
+
+### Model Compression Updates
+* Model compression PyTorch 1.4 support
+
+### Training Service Updates
+* support pai multiphase  
```

Comment:
As we've removed multiphase, can we hide this?

Reply:
fixed.